src/Analysis/Engine/Impl/DependencyResolution/PathResolverSnapshot.cs

```diff
@@ -0,0 +1,280 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.PythonTools.Analysis.Infrastructure;
+using Microsoft.PythonTools.Parsing;
+
+namespace Microsoft.PythonTools.Analysis.DependencyResolution {
+    internal partial struct PathResolverSnapshot {
+        private static readonly bool IgnoreCase = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
Looks like it can be more complicated: https://www.howtogeek.com/354220/how-to-enable-case-sensitive-folders-on-windows-10/
Or we can simply assume the "80%" case.

Reply:
I guess on Windows it is more like 0.01% since setting appears to be per folder.